UT4MasterServer.Xmpp/XmppConnection.cs

```diff
@@ -528,16 +526,6 @@ private void WriteAvailableFeatures()
 		Writer.CloseTag();
 	}
 
-	private X509Certificate SSLCertificateSelectionCallback(object sender, string targetHost, X509CertificateCollection localCertificates, X509Certificate? remoteCertificate, string[] acceptableIssuers)
-	{
-		return localCertificates[0];
-	}
-
-	private static bool SSLCertificateValidationCallback(object sender, X509Certificate? certificate, X509Chain? chain, SslPolicyErrors sslPolicyErrors)
-	{
-		return true;
-	}
-
```

Comment:
I don't think these methods should be removed. If they are unused they may be commented instead.